crates/vm/levm/src/opcode_handlers/arithmetic.rs

```diff
@@ -220,45 +220,36 @@ impl VM {
         let byte_size = current_call_frame.stack.pop()?;
         let value_to_extend = current_call_frame.stack.pop()?;
 
+        if byte_size >= U256::from(31) {
```

Comment:
```suggestion
        if byte_size > U256::from(31) {
```

Reply:
This probably doesn't change much but I saw it differed from the [execution specs](https://ethereum.github.io/execution-specs/src/ethereum/cancun/vm/instructions/arithmetic.py.html)

Reply:
[Done!](https://github.com/lambdaclass/ethrex/pull/1408/commits/c7bed6c3a2f7385e833a83a20038d7a7e4fe34ec) I didn't know that page, ty for that!